backend-services/Services/UserReportsService.py

```diff
@@ -0,0 +1,188 @@
+from flask import Blueprint
+from flask import json
+import requests
+from flask import session, request, jsonify
+from Services.UserSessionService import getAllSessions
+from datetime import datetime
+from extensions import mysql
+
+user_reports_service = Blueprint("user_reports", __name__)
+
+
+@user_reports_service.route("/get-user-session-duration")
+def userSessionDuration():
```

Comment:
this service was not migrated to this file properly. If i hit this API in postman it returns an empty instead of the expected value. I will send you a photo of the expected value in the #setup channel